### PR TITLE
Add WFH notice and a contact form under each profile

### PR DESCRIPTION
**Pull request by Qian-M20**
Can you see it this time??

### CODE REVIEW

File: example/mail.php [hunk 1]
```diff
@@ -26,57 +30,93 @@
...
-<tr>
-<td><strong>E-mail:</strong></td>
-<td>$email</td>
-</tr>
-</tbody>
-</table>
-<p>Thank you for your assistance,<br/>
-<strong>$firstName $lastName</strong></p>
-";
+// check if the form is sent for message purpose or contact update purpose
```
Comment (wosevision) [inline example/mail.php]: Could we condense this chunk a bit to remove some of the duplicated email template code? For instance, you could put the `<table>`'s contents that repeat in a separate variable, conditionally add the extra bits based on `$recipient`, and then place the contents inside the table after the fact.

```php
$tbody = "<tr>
  <td><strong>First Name:</strong></td>
  <td>$firstName</td>
  </tr>
  <tr>
  <td><strong>Last Name:</strong></td>
  <td>$lastName</td>
  </tr>
  <tr>
  <!-- etc etc -->";

$message = $recipient ? "" : "<p>Hello,
  <br/>I would like to update my contact information in the...";

$email = "$message<table><tbody>$tbody</tbody></table>";
```

File: example/mail.php [hunk 2]
```diff
@@ -26,57 +30,93 @@
...
+  <td>$subject</td>
+  </tr>
+  <tr>
+  <td><strong>Message:</strong></td>
+  <td>$message</td>
+  </tr>
+  </tbody>
+  </table>
+  ";
+}else {
```
Comment (wosevision) [inline example/mail.php]: Minor formatting nitpick: space after brace please 😸 

```php
     } else {
//    ^
//    here plz n thx

File: src/components/contact-form.component.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  <button class="close-button" aria-label="Dismiss alert" type="button" data-close>
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+<!-- FORM -->
+<form name="$ctrl.form" ng-submit="$ctrl.form.$valid && $ctrl.onSubmit({
+    $event: {
+      formData: $ctrl.formData,
+      form: $ctrl.form,
+      recipient:$ctrl.formRecipient
```
Comment (wosevision) [inline src/components/contact-form.component.html]: Another small nitpick! Space after colon

File: src/components/search-result.component.js [hunk 4]
```diff
@@ -7,11 +7,115 @@ export const SearchResultComponent = {
   },
   controller: class SearchResult {
 
-    constructor($sce) {
+    constructor($sce,$http,$httpParamSerializerJQLike) {
```
Comment (wosevision) [inline src/components/search-result.component.js]: Spaces after commas 😸 

```js
constructor($sce, $http, $httpParamSerializerJQLike) {
//               ^      ^
```

File: src/components/search-result.component.js [hunk 5]
```diff
@@ -7,11 +7,115 @@ export const SearchResultComponent = {
...
         this.result.expert.summary = $sce.trustAsHtml(this.result.expert.summary);
       }
     }
+
+
+    /**
+     * hide and show the contact form with the click of button.
+     */
+    showHide() {
+      this.IsVisible = this.IsVisible ? false : true;
```
Comment (wosevision) [inline src/components/search-result.component.js]: First letter lowercase for camelcasing, `.isVisible`

File: src/mail.php [hunk 6]
```diff
@@ -26,57 +30,93 @@
...
-<tr>
-<td><strong>E-mail:</strong></td>
-<td>$email</td>
-</tr>
-</tbody>
-</table>
-<p>Thank you for your assistance,<br/>
-<strong>$firstName $lastName</strong></p>
-";
+// check if the form is sent for message purpose or contact update purpose
```
Comment (wosevision) [inline src/mail.php]: See comment for `example/mail.php` (same)

File: src/scss/main.scss [hunk 7]
```diff
@@ -124,6 +125,10 @@ $error : pink;
 				}
 			}
 
+			.covid-note {
+				color:red;
```
Comment (wosevision) [inline src/scss/main.scss]: Space after colon

**Review by wosevision — COMMENTED** [review]
PR reviewed! Mostly just minor formatting tweaks requested.

The `mail.php` files could use a small rework; I've left comments with proposed suggestions. Let me know if you have questions! Thanks!

File: example/mail.php (hunk 1 above)
Comment (Qian-M20) [reply example/mail.php]: Good point!

File: src/components/search-result.component.js (hunk 5 above)
Comment (Qian-M20) [reply src/components/search-result.component.js]: oh well~

**Review by wosevision — APPROVED** [review]
Looks fantastic! Merged #2

**Pull request merged**